index.js

```diff
@@ -1008,31 +1053,42 @@ Multi.prototype.send_command = function (command, args, index, cb) {
     });
 };
 
-Multi.prototype.exec = Multi.prototype.EXEC = function (callback) {
+Multi.prototype.exec_transaction = function (callback) {
     var self = this;
+    var len = this.queue.length;
+    var cb;
```

Comment:
could we add a length check like https://github.com/NodeRedis/node_redis/pull/878/files#diff-168726dbe96b3ce427e7fedce31bb0bcR1170

```js
if (len === 1) {
   // no commands, only [ [ 'multi' ] ] 
  if (callback) {
    callback(null, []);
    // setImmediate(function() {
    //   callback(null, []);
    // });
  }
  return false;
}
```

there's never a reason to send `MULTI` `EXEC` with no commands!